include/TrustWalletCore/TWCoinTypeConfiguration.h

```diff
@@ -12,23 +12,29 @@
 TW_EXTERN_C_BEGIN
 
 TW_EXPORT_STRUCT
-struct TWCoinTypeConfiguration { 
+struct TWCoinTypeConfiguration {
     uint8_t unused; // C doesn't allow zero-sized struct
 };
 
+/// Returns stock symbol of coin
 TW_EXPORT_STATIC_METHOD
-TWString *_Nonnull TWCoinTypeConfigurationGetSymbol(enum TWCoinType type);
```

Comment:
It's great that the parser doesn't fail on this condition, but can you use left-pointer alignment here to be consistent with the rest of the headers?

Reply:
> It's great that the parser doesn't fail on this condition, but can you make pointer alignment here consistent with the rest of the headers?

Why? I use [.clang-format](https://github.com/TrustWallet/wallet-core/blob/master/.clang-format#L9) from this repository where: 'PointerAlignment: Left'

Reply:
Most of code pointer style is `Right`, I think we should update `.clang-format`, this pr is too much for just adding comments. Could you please open a issue for `codegen`? and just add comment in this pr